### ISSUE TITLE
Unable to alter field values by Views Bulk Operation

### ISSUE THREAD

**Issue by swirtSJW** [department-of-veterans-affairs/va.gov-cms#13583]
## Describe the defect
When trying to alter a field(s) on a node via bulk operation the apply button does not do anything.  There seems to be some javascript errors that might be preventing the form from functioning.

[Slack thread](https://dsva.slack.com/archives/C0FQSS30V/p1683068499631329)
## To Reproduce
1. Go to either of these two locations:
- generic View bulk operation https://prod.cms.va.gov/admin/content/bulk?title=&type=vet_center&moderation_state=All&taxonomy_entity_index_tid_depth=1017&page=0
- Facility specific bulk operation https://prod.cms.va.gov/admin/content/facilities/facility-status?Title=&field_facility_locator_api_id_value=&field_operating_status_facility_value=All&field_operating_status_more_info_value=&field_supplemental_status_target_id_op=or&field_supplemental_status_target_id=All&type=vet_center&moderation_state=All&workbench_access_section__section=1017
2. Select all the items in that filtered set
3. Choose the "action" of "Modifiy field values" on the former or Click "Update Operating Status" on the latter.
4. Make a change to operating status, and status moreinfo fields
5. Add revision log message
6. Click "Apply" button and watch do nothing.  Clicking multiples times has no effect.

## AC / Expected behavior
It should give a list of all facilties that will be updated and update them accordingly.

## Screenshots
![image](https://user-images.githubusercontent.com/5752113/236012926-7ea97d54-7917-40bb-ae4b-9490b370afe4.png)

 ## Additional context
Inspecting the browser console there are multiple javascript errors that are likely blocking the submission of the form.

**Comment by jilladams** [comment]
@productmike FYI. This is a general issue in the CMS, not seemingly specific to one product view. Tagged as CMS Team as well.

**Comment by kmariepat-cityfriends** [comment]
@kmariepat-cityfriends to talk to @productmike about this card for CMS, process for prioritizing work with CMS team

**Comment by kmariepat-cityfriends** [comment]
will need bulk operation for full width banners effort AND clearing COVID health status

**Comment by productmike** [comment]
@tonytaylor @edmund-dunn We need this issue to be completed in s86, so will be pulling it. We didn't get a chance to refine prior (grrr), but I'm going to assume 8 points for planning purposes. Please take a look and let me know if you agree it's well understood and able to be story pointed.

**Comment by edmund-dunn** [comment]
@productmike I identified the issue. It is NOT javascript. There are fields that are required on the form. On both example pages above the address field contains required elements. Because those elements are hidden, it generates these errors. 
![CleanShot2023-06-07at14 33 10](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/109987005/0fe8912f-37bc-4736-a8c3-c1b10d06e53b)
If, however, you check the Main location address checkbox and reveal those elements, you see the error. 
![CleanShot2023-06-07at14 42 04](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/109987005/481324ca-a565-4985-85f2-b5593e269193)
If you fill out those address elements, the Apply button works. 

After discussing with @EWashb, the solution I came up with is to remove the Required attribute from all form elements for view bulk operations like this.
